### PR TITLE
Github Actions Updated: Test on 8.5 & 8.6

### PR DESCRIPTION
**Pull request by czrpb**
Also update racket install from `Bogdanp/setup-racket@<CHANGESET>` to `Bogdanp/setup-racket@v1.9` per readme

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -16,12 +16,12 @@ jobs:
...
       matrix:
         racket-variant: ['BC', 'CS']
-        racket-version: ['8.2']
+        racket-version: ['8.5', '8.6']
 
     steps:
       - uses: actions/checkout@v3
 
-      - uses: Bogdanp/setup-racket@b0760a463ccf5e0259d5ad925be4b36b3b6e2d08
+      - uses: Bogdanp/setup-racket@v1.9
```
Comment (ErikSchierboom) [inline .github/workflows/ci.yml:24]: Could you pin this to the tag's git SHA? See https://exercism.org/docs/building/github/gha-best-practices#h-pin-actions-to-shas

**Review by ErikSchierboom — COMMENTED** [review]
One small nit

File: .github/workflows/ci.yml (hunk 1 above)
Comment (czrpb) [reply .github/workflows/ci.yml]: done!

**Comment by czrpb** [comment]
looks like it is the same changeset and blame shows an automated process did the changeset change 1day after the version was updated. so basically, the only change in this PR is to use 8.5 and 8.6 for testing.

**Comment by czrpb** [comment]
ASIDE: I find exercism's logic wrt tags strange.

i wonder if they would advocate github to remove the tags features as basically they are saying it is a security risk!! 😉

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**